tests/ruby/tests.bats

```diff
@@ -8,6 +8,7 @@ setup() {
     rm -rf /home/application/current && mkdir /home/application/current
     chown ubuntu /home/application/current
     export CURRENT_DIR=/home/application/current
+    export PATH=/home/application/ruby/bin:${PATH}
```

Comment:
I had to add this, otherwise the tests (and the deploy script) would be using `ruby`, `gem` and other binaries from `/usr/local/bin` instead of those from the build pack.